src/backer/cardaning.py

```
@@ -64,282 +52,216 @@ class PointRecord:
-            "type": type.value,
-            "keri_raw": submitting_items,
+        signedTxMetadata = {
+            "id": str(signedTx.id),
+            "type": txType.value,
+            "batch": eventsInBatch,
             "tip": self.tipHeight,
         }
         utxoIds = [f"{str(utxo.input.transaction_id)}#{utxo.input.index}" for utxo in utxos]
 
-        dbConfirming.pin(keys=submitting_trans["id"], val=json.dumps(submitting_trans).encode('utf-8'))
-        self.dbConfirmingUtxos.pin(keys=submitting_trans["id"], vals=utxoIds)
+        confirmingTxs.pin(keys=signedTxMetadata["id"], val=json.dumps(signedTxMetadata).encode('utf-8'))
+        self.dbConfirmingUtxos.pin(keys=signedTxMetadata["id"], vals=utxoIds)
 
         # Submit transaction
         try:
-            self.context.submit_tx_cbor(submitting_tx_cbor)
-
-            # Remove submitted events from queue and add them into published
-            for event in submitting_items:
-                event = event.encode('utf-8')
-                self.addToPublished(event, type)
-                self.removeFromQueue(event, type)
-
-            logger.info(f"Submitted tx: {submitting_trans}")
+            self.context.submit_tx_cbor(bytes(signedTx.to_cbor()))
+            logger.info(f"Submitted tx: {signedTxMetadata}")
         except Exception as e:
             logger.critical(f"ERROR: Submit tx: {e}")
-            # Add back to queue
-            for keri_item in submitting_trans['keri_raw']:
-                self.addToQueue(keri_item.encode('utf-8'), type)
 
             # Remove from confirming
-            self.dbConfirmingUtxos.rem(keys=submitting_trans["id"])
-            dbConfirming.rem(submitting_trans["id"])
-
-    def getConfirmingTrans(self, transId):
-        tx = self.schemadbConfirming.get(transId)
-
-        if not tx:
-            tx = self.keldbConfirming.get(transId)
-
-        return tx
-
-    def rollbackBlock(self, rollBackSlot, type:CardanoType):
-        try:
```

This try catch was a mistake! It meant we would move onto the next block in the crawler. So better to let the error throw to stop crawling